AppCheckCore.podspec

```diff
@@ -22,7 +22,7 @@ Pod::Spec.new do |s|
   tvos_deployment_target = '12.0'
   watchos_deployment_target = '6.0'
 
-  s.swift_version = '5.3'
+  s.swift_version = '5.7.1'
```

Comment:
Will you change this to 5.5 separately?

Reply:
Yes, will do in another PR (along with `Package.swift`).